feature-toggles/feature-toggles-api/src/main/java/com/duckduckgo/feature/toggles/api/FeatureToggles.kt

```diff
@@ -484,26 +487,12 @@ internal class ToggleImpl constructor(
                 cohorts[randomIndex.sample()]
             }.getOrNull()
         }
-        fun containsAndMatchCohortTargets(targets: State.Target?): Boolean {
-            return targets?.let {
-                targets.localeLanguage?.let { targetLanguage ->
-                    val deviceLocale = localeProvider.invoke()
-                    if (deviceLocale?.language != targetLanguage) {
-                        return false
-                    }
-                }
-                targets.localeCountry?.let { targetCountry ->
-                    val deviceLocale = localeProvider.invoke()
-                    if (deviceLocale?.country != targetCountry) {
-                        return false
-                    }
-                }
-                return true
-            } ?: return true // no targets mean any target
+        fun containsAndMatchCohortTargets(targets: List<State.Target>): Boolean {
+            return callback?.matchesToggleTargets(targets) ?: true
         }
 
         // In the remote config, targets is a list, but it should not be. So we pick the first one (?)
-        if (!containsAndMatchCohortTargets(targets.firstOrNull())) {
```

Comment:
This was certainly wrong before, but we never added more than one target so far so 🤷